nginx-directory/config.libsonnet

```diff
@@ -1,14 +1,38 @@
 {
   _config+:: {
-    // oauth2-proxy
-    oauth_enabled: false,
+    namespace: error 'namespace required',
+    cluster_dns_suffix: error 'cluster_dns_suffix required',
+
+    admin_services+: [
+      // an entry should look like this:
+      //  {
+      //    # Required
+      //    title: 'Prometheus',
+      //    path: 'prometheus',
+      //    url: 'http://prometheus.default.svc.cluster.local/prometheus/',
+      //
+      //    # Optional
+      //    params: '',
+      //    read_timeout: '60',
+      //    send_timeout: '60',
+      //    redirect: false,
+      //    allowWebsockets: false,
+      //    subfilter: false,
```

Comment:
could usefully explain what `subfilter` does!

Reply:
Yes, someone should write docs. :innocent: